src/Sarif.Viewer.VisualStudio/SarifExplorerWindow.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using System;
+using System.Runtime.InteropServices;
+using System.Windows;
+using Microsoft.Sarif.Viewer.ErrorList;
+using Microsoft.Sarif.Viewer.Tags;
+using Microsoft.Sarif.Viewer.Views;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.Sarif.Viewer.Models;
```

Comment:
ordering

Reply:
Seriously, turn on the analyzers in this repo :)

---
In reply to: [502563038](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502563038) [](ancestors = 502563038)